warden/x/warden/keeper/shield.go

```diff
@@ -16,33 +17,77 @@ type WardenShieldExpander struct {
 }
 
 func (w WardenShieldExpander) Expand(goCtx context.Context, ident *ast.Identifier) (*ast.Expression, error) {
+	if strings.HasPrefix(ident.Value, "analyzer.") {
```

Comment:
yeah I suppose it makes more sense, in the full intent definition it would look like this:
```
warden.analyzers.warden123123123.foo
```

Reply:
done

Reply:
> yeah I suppose it makes more sense, in the full intent definition it would look like this:
> 
> ```
> warden.analyzers.warden123123123.foo
> ```

Sorry, I still do not understand, you provide example with single analyzer-contract. What if user need to provide multiple contract addresses and own keys set for every contract?

Reply:
imagine I want to write an intent for checking if an ETH transaction is transferring more than 10 WEI, I can write it like that:

```
warden.analyzers.warden12312313.value > 10
```

where `warden12312313` is the address of the Ethereum analyzer smart contract.

You can also use more than one analyzer in a single intent definition:

```
warden.analyzers.warden12312313.value > 10 ||
warden.analyzers.wardenXXXXXXXX.some_foo_var
```

does that answer your question?

Reply:
ah yeah! This is the list of analyzers that you want to execute, because I thought maybe the user has a big list of "or"s with many analyzers but already knows which one must be executed to fulfill the intent, by doing less invokations the user saves some gas 

should've mention this somewhere, sorry for the confusion 😅